app/controllers/IndexController.scala

```diff
@@ -75,4 +76,15 @@ class IndexController @Inject()(
         Future.successful(Redirect(navigator.nextPage(IndexPage, NormalMode)(initialAnswers)))
     }
   }
+  def getApplicationsAndRulings(page: Int): Action[AnyContent] = identify.async { implicit request =>
+    request.eoriNumber match {
+      case Some(eori: String) =>
+        service.getCases(eori, applicationStatuses, SearchPagination(page), Sort()) flatMap { pagedResult =>
+          successful(Ok(account_dashboard_statuses(appConfig, pagedResult)))
+        }
+
+      case None => successful(Redirect(routes.BeforeYouStartController.onPageLoad()))
```

Comment:
I think that this will need to set the `initialAnswers` like the `getRulings` and `getApplications` functions do

Reply:
I have added the changes. Thanks David